models.py

```diff
@@ -315,6 +315,115 @@ def bounding_box_query(ne_lat, ne_lng, sw_lat, sw_lng):
             .order_by(desc(DiscussionMarker.created))
         return markers
 
+class Involved(Base):
+    __tablename__ = "involved"
+    id = Column(Integer, primary_key=True)
+    accident_id = Column(Integer, ForeignKey("markers.id"))
+    accident_year = Column(Integer)
+    month = Column(Integer)
+    involved_type = Column(Integer)
+    license_aquiring_date = Column(Integer)
+    age_group = Column(Integer)
+    sex = Column(Integer)
+    car_type = Column(Integer)
+    safety_measures = Column(Integer)
+    home_city = Column(Integer)
+    injury_severity = Column(Integer)
+    injured_type = Column(Integer)
+    injured_position = Column(Integer)
+    population_type = Column(Integer)
+    home_district = Column(Integer)
+    home_nafa = Column(Integer)
+    home_area = Column(Integer)
+    home_municipal_status = Column(Integer)
+    home_residence_type = Column(Integer)
+    file_type = Column(Integer)
+
+    def serialize(self):
+        return {
+            "id": self.id,
+            "accident_id": self.accident_id,
+            "accident_year": self.accident_year,
+            "month": self.month,
+            "involved_type": self.involved_type,
+            "license_aquiring_date": self.license_aquiring_date,
+            "age_group": self.age_group,
+            "sex": self.sex,
+            "car_type": self.car_type,
+            "safety_measures": self.safety_measures,
+            "home_city": self.home_city,
+            "injury_severity": self.injury_severity,
+            "injured_type": self.injured_type,
+            "injured_position": self.injured_position,
+            "population_type": self.population_type,
+            "home_district": self.home_district,
+            "home_nafa": self.home_nafa,
+            "home_area": self.home_area,
+            "home_municipal_status": self.home_municipal_status,
+            "home_residence_type": self.home_residence_type,
+            "file_type": self.file_type
```

Comment:
`file_type` don'y need a column, it's just concatenated to the original accident id.
We must concatenate them in order to combine both `Marker` and `Involved` tables correctly.